src/payload/orderPayloadBuilder.ts

```diff
@@ -0,0 +1,69 @@
+import OrderPayload from "./orderPayload";
+import { Sender } from "../models/sender";
+import { Recipient } from "../models/recipient";
+
+export default class OrderPayloadBuilder {
+    quotationId: string | undefined;
+
+    sender: Sender | undefined;
+
+    recipients: Recipient[] | undefined;
+
+    fleetOption?: string;
```

Comment:
As `fleetOption` is not mentioned in our documentation, it would be better to not specify it in our SDK. TBC with Sang Ha

Reply:
sure, then I will leave it here until the confirmation?